### PR TITLE
Updating SIG Scalibility readME

### LINKED ISSUE #3798: SIG Scalability out of date
The https://github.com/kubernetes/community/blob/master/sig-scalability/README.md mentions weekly meetings but based on the agenda and youtube playlist it seems this is no longer the case, with notes of one meeting perhaps held in 2019 and no videos uploaded since 2017.

### PR DESCRIPTION
**Pull request by vishakhanihore**
Fixes #3798

### CODE REVIEW

File: sig-scalability/README.md [hunk 1]
```diff
@@ -12,11 +12,6 @@ SIG Scalability is responsible for defining and driving scalability goals for Ku
 
 The [charter](charter.md) defines the scope and governance of the Scalability Special Interest Group.
 
-## Meetings
```
Comment (wojtek-t) [inline sig-scalability/README.md]: Why do you remove it? Meetings are happening biweekly (we just don't have recordings - hopefully the next one will be recorded). Agenda also started to be up-to-date in the last couple months.

**Comment by wojtek-t** [comment]
@mm4tt - FYI

/hold

**Comment by vishakhanihore** [comment]
@wojtek-t Please see the updated documentation.

**Comment by mrbobbytables** [comment]
I think the parent issue (https://github.com/kubernetes/community/issues/3798) is out of date. The current information in there looks accurate.

Also, as an FYI the SIG README's are auto-generated and should not be updated by hand with the exception of the content between `<!-- BEGIN CUSTOM CONTENT -->`. 😬

Their information is sourced from entries in [sigs.yaml](https://github.com/kubernetes/community/blob/master/sigs.yaml), and our docs generator uses a template to update both the README and the [sig-list.md](https://github.com/kubernetes/community/blob/master/sig-list.md) doc. For a bit more information, check out the [docs generator README](https://github.com/kubernetes/community/tree/master/generator#sig-doc-builder).

**Comment by vishakhanihore** [comment]
@mrbobbytables Thanks for the info, No worries I will close the PR.

**Pull request closed without merging**